--- old/linux-gpib-kernel/drivers/Makefile.am	2018-04-09 14:53:49.000000000 -0400
+++ new/linux-gpib-kernel/drivers/Makefile.am	2018-06-11 19:18:56.370076564 -0400
@@ -14,10 +14,8 @@
 
 -include $(top_srcdir)/util/linux_flags/flags
 
-#ignore errors from kernel's make modules in all-local, to avoid failure of make install
-#on nfs filesystem with root_squash
 all-local:
-	-$(MAKE) -C $(LINUX_SRCDIR) V=1 modules\
+	$(MAKE) -C $(LINUX_SRCDIR) V=1 modules\
 		CC="$(LINUX_CC) -I@abs_top_srcdir@ -I@abs_top_srcdir@/compat/include" \
 		CONFIG_GPIB_ISA="$(ENABLE_ISA)" \
 		SUBDIRS="$(GPIB_ABS_MODULE_SUBDIR)"
