--- old/linux-gpib-user/configure.ac	2022-06-06 16:20:34.432280304 -0400
+++ new/linux-gpib-user/configure.ac	2022-06-06 16:28:56.968945905 -0400
@@ -205,10 +205,6 @@
 	language/php/TESTS/Makefile \
 	language/python/Makefile \
 	language/tcl/Makefile \
-	usb/Makefile \
-	usb/ni_usb_gpib/Makefile \
-	usb/agilent_82357a/Makefile \
-	usb/lpvo_usb_gpib/Makefile \
 
 ])
 AC_OUTPUT

--- old/linux-gpib-user/Makefile.am	2022-06-06 16:20:34.502280106 -0400
+++ new/linux-gpib-user/Makefile.am	2022-06-06 16:29:22.492879236 -0400
@@ -7,7 +7,7 @@
 #   the Free Software Foundation; either version 2 of the License, or
 #   (at your option) any later version.
 
-SUBDIRS = include lib drivers examples test doc language usb
+SUBDIRS = include lib drivers examples test doc language 
 
 EXTRA_DIST = etc util m4 README.HAMEG README.hp82335 bootstrap
 
